templates/triggers/trigger_schedule_in_batch.haml

```diff
@@ -0,0 +1,153 @@
+            window.document.location.href = jqXHR.getResponseHeader('REDIRECT');
+          }
+        }
+      });
+    }
+
+    function buildFormData(form) {
+      return form.serializeArray().reduce(function (acc, item) {
+          let value = item.value;
+          if (item.name === 'omnibox' && value) {
+            value = acc['omnibox'] || [];
+            value.push(item.value);
+          }
+          acc[item.name] = value;
+          return acc;
+        }, {});
+    }
+
+    document.addEventListener("DOMContentLoaded", function(){
+      var repeatPeriod = document.querySelector("#id_schedule_repeat_period");
+      if (repeatPeriod) {
+        repeatPeriod.addEventListener("change", function(event){
+          var currentRepeat = event.currentTarget.values[0].value;
+          if (currentRepeat === 'W') {
+            document.querySelector(".weekly-repeat-options").classList.remove("hidden");
+          } else {
+            document.querySelector(".weekly-repeat-options").classList.add("hidden");
+          }
+        });
+      }
+
+      let inBatchConfirmation = document.querySelector("#bulk-schedule-confirmation");
+      const section =  $('#id-trigger-schedule-in-batch');
+      const form = section.find('form');
+      const flowOptionList = document.querySelector('#id_schedule_flow').querySelectorAll('[name]');
+      const flowById = {};
+      flowOptionList.forEach(function (node) {
+        flowById[node.getAttribute('value').toString()] = node.getAttribute('name');
+      });
+
+      form.off("submit").on("submit", function (e) {
+        e.preventDefault();
+
+        const formDataByName = buildFormData(form)
+
+        const modalMessage = $('#bulk-schedule-confirmation').find('div.p-6');
+        const flowName = flowById[formDataByName.flow];
+        const selectedGroups = (formDataByName.omnibox || []).length;
+        const message = messageTemplate
+          .replace('{flowName}', flowName || '--')
+          .replace('{groupCount}', String(selectedGroups))
+          .replace('{batchInterval}', formDataByName.batch_interval)
+
+        modalMessage.text(message);
+        inBatchConfirmation.classList.remove("hide");
+        inBatchConfirmation.open = true;
+      });
+
+      inBatchConfirmation.addEventListener("temba-button-clicked", function(event){
+        if (!event.detail.button.secondary) submitFormData(form);
+        inBatchConfirmation.classList.add("hide");
+        inBatchConfirmation.open = false;
+      });
+    });
+
+
+-block fields
+  #trigger_flow
+    - render_field 'flow'
+  #trigger_omnibox
+    - render_field 'omnibox'
+  #schedule-options
+    -render_field 'start_datetime'
+  #recurrence
+    .repeat-period
+      -render_field 'repeat_period'
+    .weekly-repeat-options{class:'{% if form.repeat_period.value == "W" %}{% else%}hidden{%endif%}'}
+      -render_field 'repeat_days_of_week'
+  #batch_interval
+    -render_field 'batch_interval'
+  %h5.embedded-data-schedule.hidden
```

Comment:
The embedded data (flow parameters) fields are not being displayed when the flow selected has @trigger.params variables, for example, if it has `@trigger.params.test` in some send message flow step. Keyword and Schedule triggers have this behavior (and are working), so you can check it out there.

![Screen Shot 2021-08-09 at 22 06 30](https://user-images.githubusercontent.com/5198870/128793039-add5cf08-351f-4c9c-9335-cbdd36947c0d.png)